helm_deploy/prisoner-offender-search/values.yaml

```diff
@@ -44,11 +44,20 @@ generic-service:
     requests:
       memory: 856Mi
 
-  livenessProbe:
-    failureThreshold: 20
+  startupProbe:
+    httpGet:
+      path: /health/liveness
+      port: http
+    initialDelaySeconds: 30
+    periodSeconds: 10
+    timeoutSeconds: 10
+    failureThreshold: 30
```

Comment:
This gives the application 30s + (30x10s) = 5m30s to startup. I arrived at this figure because when we've experienced startup problems relying on just the liveness probe we found that 30s + (20x10s) = 3m50s wasn't long enough. And I don't think having a startup that is a bit too long will hurt us.